grafana-plugin/src/containers/IntegrationForm/HowTheIntegrationWorks.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
```

Comment:
How do we differentiate between components and containers? I thought that something that is not connected to store and is not making any backend requests can be considered a component (even though it's related to our domain - integration)